src/Security/SessionAPIAuthenticator.php

```diff
@@ -208,7 +213,12 @@ public function getUser($credentials, UserProviderInterface $userProvider)
      */
     public function checkCredentials($credentials, UserInterface $user)
     {
-        return true;
+        if (!array_key_exists('session', $credentials)) {
+            throw AuthenticationServiceException::sessionTokenNotFound();
```

Comment:
is it correct to throw AuthenticationServiceException outside of AuthenticationService scope ?